src/backend/src/services/change-requests.services.ts

```diff
@@ -534,12 +543,17 @@ export default class ChangeRequestsService {
     });
 
     const project = createdCR.wbsElement.workPackage?.project || createdCR.wbsElement.project;
-    if (project?.team) {
+    const teams = project?.teams;
+    if (!teams) {
+      throw new HttpException(400, 'need a team');
```

Comment:
1. Bad error msg
2. Do the same as above, I wanna avoid changing a bunch of logic in this PR, since it already has so much in it, if it has teams, send the slack msg, if it doesn't do nothing